shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -381,6 +383,68 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
     sendViewportMetricsToFlutter();
   }
 
+  // TODO(garyq): Add support for notch cutout API
+  // Decide if we want to zero the padding of the sides. When in Landscape orientation,
+  // android may decide to place the software navigation bars on the side. When the nav
+  // bar is hidden, the reported insets should be removed to prevent extra useless space
+  // on the sides.
+  enum ZeroSides {
+    NONE,
+    LEFT,
+    RIGHT,
+    BOTH
+  }
+
+  ZeroSides calculateShouldZeroSides() {
```

Comment:
What visibility should this and `ZeroSides` have? I'd probably shoot for private rather than expose this utility method until we have people asking us to do so.

Since this class is non-final, and this is used from `onApplyWindowInsets` below, we should probably require that any overrides call `super.onApplyWindowSets()` (if that's reasonable -- I admit I didn't read through all the code) or expect that one day someone will want us to expose this publicly.

Reply:
Ahh yes, these should definitely be private. Not sure why I didn't make them private in the original code.